static/sass/_pattern_expandable-section.scss

```diff
@@ -0,0 +1,21 @@
+@mixin ubuntu-p-expandable-section {
```

Comment:
Not sure I understand this - it looks like it's aliasing existing Vanilla `p-accordion` classes to `p-expandable-section` classes, without adding anything custom. I think it might be that this class is being used so it can be referenced in the [js above](#diff-4877968da7344af822e0e9e508c5634c0572082f0319df7e7cf014d131855b13), without interfering with the existing accordion script, is that right?

Reply:
Yeah that's right. The styling needs to be the same as the `p-accordion` but the functionality different. As accordions aren't built to have more than one section open at a time, (which is what we need) I needed to write the javascript for this

Reply:
@sowasred2012 I've made the suggestions changes - not sure on the data attribute name but can't think of anything to explain it better if you have any suggestions!